src/snowflake/cli/_plugins/nativeapp/exceptions.py

```diff
@@ -111,3 +111,15 @@ class NoEventTableForAccount(ClickException):
 
     def __init__(self):
         super().__init__(f"{self.__doc__}\n\n{self.INSTRUCTIONS}")
+
+
+class DistributionAttributeNotFoundError(ClickException):
```

Comment:
We can probably make that more generic. Something like "ObjectPropertyNotFound" or "UnexpectedApplicationPackageProperties" ?